Clinical-Genomics/scout
# Allow users to customize institute settings

fix #988 

This PR introduces a page where an institute might be customized at this level:
- Display name
- Sanger recipients list
- Coverage cutoff
- Frequency cutoff
- Phenotype groups
- Patients cohorts
- Collaborating institutes

**How to test**:
1. Probably a good idea to test this on clinical-db stage.
1. Install the branch and go to "Institutes" page. Note that an edit button is present for each institute the user has permission to change (user shouldn't be able to modify institutes he or she is not allowed to edit)
1. Edit institute-related fields

**Expected outcome**:
The functionality should be working and changes should be committed to database.

**Review:**
